addons/website_slides_survey/models/survey_survey.py

```diff
@@ -34,7 +34,8 @@ def _unlink_except_linked_to_course(self):
         if certifications:
             certifications_course_mapping = [_('- %s (Courses - %s)', certi.title, '; '.join(certi.slide_channel_ids.mapped('name'))) for certi in certifications]
             raise ValidationError(_(
-                'Any Survey listed below is currently used as a Course Certification and cannot be deleted:\n%s',
+                'Uh-oh! You can’t delete a survey used as a Course Certification! Otherwise students might think diplomas just grow on trees.\n'
+                'The courses that need it:\n%s',
```

Comment:
```suggestion
                'The courses that need it are:\n%s',
```

Reply:
Please remove the tree part, makes no sense. You don't HAVE to find a bad joke, especially this flow is accessible only to survey managers.